packages/devtools-kit/src/ctx/hook.ts

```diff
@@ -295,6 +298,25 @@ export function createDevToolsCtxHooks() {
           resolve()
         }, DevToolsV6PluginAPIHookKeys.GET_INSPECTOR_STATE)
       })
+
+      // Expose instance data to window
+      // Copied from https://github.com/vuejs/devtools/blob/f03590025b0b4910cf539531c91384be51a8f8fa/packages/app-backend-core/src/component.ts#L57-L72
+      const instance = getComponentInstance(activeAppRecord.value, _payload.nodeId)
```

Comment:
I think we can create `core/vm/index.ts` and export `exposeInstanceToWindow` function to use it.